RIT-MDRC/Catbot
# Feat/arduino pressure monitor

# Feature
* Arduino will turn on the compressor and pressurize until max pressure is reached
* Arduino will also serial write the pressure it is reading
* Arduino will open the valve to bend the leg when only the valve button is pressed
* Arduino will also close the valve if the valve is opened for 5 seconds

# How to Test
* upload the code using the arduino IDE if not uploaded on the arduino already
* Check to make sure the following is satisfied
  - The valve next to the compressor is closed
  - The leg is properly attached
* Turn on the board using the big switch
* Open the serial monitor (pressure reading should be seen)
* Wait for the system to pressurize
* Once pressurized check to make sure 82 psi is reached
* Press on the valve button
* the muscle should inflate 
* the compressor should turn back on if the lower than ideal pressure
* Once the muscle is inflated for 5 seconds the valve will close
* pressing on the valve switch past this point will cause the muscle to inflate again
* When the muscle is inflated and both the compressor button and the valve switch is pressed then the valve should close 
* Only pressing on the compressor button will cause the compressor to turn and overwrite the system

merged with #18
